test/e2e/template/busyboxds.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: busybox
+  namespace: default
+spec:
+  template:
+    metadata:
+      labels:
+        component: busybox
+    spec:
+      hostNetwork: true
+      containers:
+      - name: busybox
+        image: alpine:3.8
+        stdin: true
+        securityContext:
+          privileged: true
```

Comment:
do we need privileged for this ? Isn't hostNetwork: true enough to check the iptables existence ?

Reply:
Yeah, without privileged the iptables table won't initialize.